### PR TITLE
Viite 492: hower info popup for roadaddress in project link action selection mode

### PR DESCRIPTION
**Pull request by oappicgi**
hower

### CODE REVIEW

File: viite-UI/src/view/ProjectLinkLayer.js [hunk 1]
```diff
@@ -212,6 +212,65 @@
     //This will control the double click zoom when there is no selection that activates
     map.on('dblclick', zoomDoubleClickListener);
 
+    var infoContainer = document.getElementById('popup');
+    var infoContent = document.getElementById('popup-content');
+
+    var overlay = new ol.Overlay(({
+      element: infoContainer
+      //autoPan: false,
```
Comment (pedrosag) [inline viite-UI/src/view/ProjectLinkLayer.js]: Is this one needed?

**Comment by Campanha** [comment]
this is a nice feature!

File: viite-UI/src/view/ProjectLinkLayer.js (hunk 1 above)
Comment (oappicgi) [inline viite-UI/src/view/ProjectLinkLayer.js]: yea, we tested that with Jani and forgot it there. if enabled popup would in cases where it is near corners of the map  push map in direction it appears.

**Comment by tvenhola** [comment]
Looks good

**Pull request merged**